python/dgl/graphbolt/impl/sampled_subgraph_impl.py

```diff
@@ -145,7 +174,20 @@ def exclude_edges(
         should be a pair of tensors representing the edges to exclude. If
         sampled subgraph is heterogeneous, then `edges` should be a dictionary
         of edge types and the corresponding edges to exclude.
-
+    exclude_mode : Union[str, callable], optional
+        Whether and how to exclude dependencies related to the provided edges
+        in the argument ``edges``. Possible values are
+        - If None, for excluding only provided ``edges``.
+        - If "reverse", for excluding not only the provided ``edges`` but aslo
+          their reverse edges. In the case of a homogeneous graph, reverse
+          edges refer to edges with inverted source and destination node IDs.
+          In heterogeneous graphs, this encompasses reversing both node IDs
+          and types, as specified by the ``reverse_types`` argument.
+        - If User-defined exclusion rule. It is a callable with edges in the
```

Comment:
It is not necessary to allow callable, `exclude_edges(subgraph, edges, callable)` is equivalent to `exclude_edges(subgraph, callable(edges))`.